apps/openchallenges/challenge-service/src/main/resources/db/challenges.csv

```diff
@@ -477,11 +477,11 @@
-"479","vqa-answertherapy-2024","VQA-AnswerTherapy 2024","Grounding all answers for each visual question","Visual Question Answering (VQA) is a task of predicting the answer to a question about an image. Given that different people can provide different answers to a visual question, we aim to better understand why with answer groundings. To achieve this goal, we introduce the VQA-AnswerTherapy dataset, the first dataset that visually grounds each unique answer to each visual question. We offer this work as a valuable foundation for improving our understanding and handling of annotator differences. This work can inform how to account for annotator differences for other related tasks such as image captioning, visual dialog, and open-domain VQA (e.g., VQAs found on Yahoo!Answers and Stack Exchange). This work also contributes to ethical AI by enabling revisiting how VQA models are developed and evaluated to consider the diversity of plausible answer groundings rather than a single (typically majority) one.","https://evalai.s3.amazonaws.com/media/logos/e63bc0a0-cd35-4418-b32b-4ef2b9c61ce2.png","https://eval.ai/web/challenges/challenge-page/1910/overview","upcoming","16","","2024-01-30","\N","2023-12-12 22:41:48","2023-12-12 23:20:41"
+"479","vqa-answertherapy-2024","VQA-AnswerTherapy 2024","Grounding all answers for each visual question","Visual Question Answering (VQA) is a task of predicting the answer to a question about an image. Given that different people can provide different answers to a visual question, we aim to better understand why with answer groundings. To achieve this goal, we introduce the VQA-AnswerTherapy dataset, the first dataset that visually grounds each unique answer to each visual question. We offer this work as a valuable foundation for improving our understanding and handling of annotator differences. This work can inform how to account for annotator differences for other related tasks such as image captioning, visual dialog, and open-domain VQA (e.g., VQAs found on Yahoo!Answers and Stack Exchange). This work also contributes to ethical AI by enabling revisiting how VQA models are developed and evaluated to consider the diversity of plausible answer groundings rather than a single (typically majority) one.","https://evalai.s3.amazonaws.com/media/logos/e63bc0a0-cd35-4418-b32b-4ef2b9c61ce2.png","https://eval.ai/web/challenges/challenge-page/1910/overview","active","16","","2024-01-30","2199-12-26","2023-12-12 22:41:48","2024-01-31 23:05:00"
```

Comment:
Wow, I won't be around long enough to see this challenge end, lol.

Reply:
Who knows, with today's technology, we could still be around 😆